R/wiipoc.R

```diff
@@ -13,8 +13,10 @@ jupyter.widget.Button$set("private", "before_comm_open", function() {
 })
 
 jupyter.widget.Box$set("private", "children_", list())
+jupyter.widget.Accordion$set("private", "children_", list())
```

Comment:
We need to add the `children_` private field so that the R object holds references to the widgets themselves, just like `box`